src/nvim/os/win_defs.h

```diff
@@ -6,16 +6,22 @@
 #define TEMP_DIR_NAMES {"$TMP", "$TEMP", "$USERPROFILE", ""}
 #define TEMP_FILE_PATH_MAXLEN _MAX_PATH
 
-// Defines needed to fix the build on Windows:
-// - DFLT_DIR
-// - DFLT_BDIR
-// - DFLT_VDIR
-// - EXRC_FILE
-// - VIMRC_FILE
-// - SYNTAX_FNAME
-// - DFLT_HELPFILE
-// - SYS_VIMRC_FILE
-// - SPECIAL_WILDCHAR
+// Windows system-dependent file names
+#ifndef SYS_VIMRC_FILE
+# define SYS_VIMRC_FILE "$VIM\\sysinit.vim"
+#endif
+#ifndef DFLT_HELPFILE
+# define DFLT_HELPFILE  "$VIMRUMTIME\\doc\\help.txt"
+#endif
+#ifndef SYNTAX_FNAME
+# define SYNTAX_FNAME   "$VIMRUNTIME\\syntax\\%s.vim"
+#endif
+#ifndef EXRC_FILE
+# define EXRC_FILE      "_exrc"
+#endif
+#ifndef VIMRC_FILE
+# define VIMRC_FILE     "_nvimrc"
```

Comment:
:+1:

We can then move these (`EXRC_FILE`, `VIMRC_FILE`) to `globals.h` too yes?

Since they would then be defined the same as what's currently in `unix_defs.h`.